astropy/stats/jackknife.py

```diff
@@ -74,16 +74,16 @@ def jackknife_stats(data, statistic, confidence_level=0.95):
 
     Returns
     -------
-    estimate : numpy.float64 or numpy.ndarray
+    estimate : `~numpy.double` or `~numpy.ndarray`
```

Comment:
This needs review from the stats maintainer. Is it really `double` or `float64`?

Reply:
https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.float64

Numpy float64 is an alias of numpy.double. I had trouble linking to float64, but not to double.

Reply:
I think this should be changed to simply `float`.  In Numpy 1.2.0, numpy's float is a deprecated alias for the builtin `float`.

Reply:
How about `number`: Any numeric type. eg float or int or any of the :class:`~numpy.number`.

Maybe the glossary term should specifically exclude complex?

Reply:
See #11555